cmd/mt-fakemetrics/README.md

```diff
@@ -42,14 +42,56 @@ The speed is 100x what it would be if it were realtime (so a rate of 4x100x100=4
 mt-fakemetrics backfill --kafka-mdm-addr localhost:9092 --offset 5h --period 10s --speedup 100 --orgs 4 --mpo 100
 ```
 
-# Outputs
+## Value Policies
+
+You can specify a Value Policy with the `--value-policy` flag. Currently the following Value Policies are supported:
+
+* single
+* multiple
+* timestamp
```

Comment:
why take away the ability to use random values?

Reply:
The default behavior with no policy specified is still the same, it will generate a random float. I am updating the documentation to reflect this.